src/django/api/management/commands/post_deployment.py

```
@@ -8,6 +8,6 @@ class Command(BaseCommand):
             'post-deployment tasks.')
 
     def handle(self, *args, **options):
```

Ruff (0.15.1)

[warning] Unused method argument: `args`

(ARG002)